server/pages/index.js

```diff
@@ -183,7 +211,10 @@ function getDefaultContext( request ) {
 		isDebug,
 		badge: false,
 		lang: config( 'i18n_default_locale_slug' ),
-		jsFile: 'build',
+		entrypoint: getAssets().entrypoints.build.assets.filter(
+			asset => ! asset.startsWith( 'manifest' )
+		),
+		manifest: getAssets().manifests.manifest,
```

Comment:
this is strange.  while the assets writer supports multiple manifest, this page doesn't.

Does desktop need any changes here?